datalake/src/main/java/com/sequenceiq/datalake/flow/SdxReactorFlowManager.java

```diff
@@ -118,6 +118,9 @@ public FlowIdentifier triggerSdxResize(Long sdxClusterId, SdxCluster newSdxClust
         boolean performRestore = sdxBackupRestoreService.shouldSdxRestoreBePerformed(
                 newSdxCluster, entitlementService.isDatalakeBackupOnResizeEnabled(ThreadBasedUserCrnProvider.getAccountId())
         );
+        if (!performBackup) {
+            sdxBackupRestoreService.checkExistingBackup(newSdxCluster, userId);
```

Comment:
Isn't this call going to fail? We don't have the DR service in the QE environment. We should have a check here whether the service is configured or not.

Reply:
The `checkExistingBackup` method contains the following condition:
```java
        if (!datalakeDrConfig.isConfigured()) {
            return;
        }
```

Therefore the QE env will not failed on this.

Reply:
Ah, missed that, thank you.